lambdas/src/handlers/script-builder.ts

```diff
@@ -0,0 +1,38 @@
+import * as _ from "lodash";
+import {SecretsManager} from "./secrets-manager";
+import {User} from "./user";
+
+export class ScriptBuilder {
```

Comment:
Are all these SQL scripts are specific to MySQL or can they also be run by PostgreSQL?